.github/workflows/driver_ci.yaml

```diff
@@ -167,7 +167,7 @@ jobs:
       - name: Get the operator github branch
```

Comment:
```suggestion
      - name: Get the driver github branch
```
No?

Reply:
Resolving this comment since we need the operator's branch here (see script below for more information)